templates/donations/email_donation_request.txt

```diff
@@ -18,4 +18,7 @@ Why should you donate to Freesound?
 In case you want to donate, please use the link below:
 
 {% absurl 'donate' %}
+
+If you don't want to receive these kind of notifications anymore you can turn them off at {% absurl "accounts-email-settings" %}
```

Comment:
with the templates, can we have an external file that we include so that we don't have to write the same thing in all email templates?

Reply:
Yes, done in 42331de9c5b276847281328719619e4660ff911e